recipes/gpflow/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "1.1.1" %}
+
+package:
+  name: gpflow
+  version: {{ version }}
+
+source:
+  fn: GPflow-{{ version }}.tar.gz
+  url: https://github.com/GPflow/GPflow/archive/{{ version }}.tar.gz
+  sha256: e92737e9488360aca754c24da434118c625aa9737bc22d2a58e41caf7c3f9122
+
+build:
+  number: 0
+  skip: True  # [py<35]
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - scipy
+    - pandas
+    - tensorflow
+    - multipledispatch
+    - pytest
+
+test:
+  imports:
+    # Skip the import test on Linux, see tensorflow recipe
+    - gpflow  # [not linux]
+
+about:
+  home: https://github.com/GPflow/GPflow/
+  license: Apache 2.0
+  license_family: Apache
+  license_file: LICENSE
+  summary: Gaussian process methods in tensorflow
+
+extra:
+  recipe-maintainers:
+    - cxhernandez
```

Comment:
@cxhernandez, happy to be a maintainer of the gpflow conda-forge recipe?

Reply:
Sure!  :+1: